benefits/eligibility/views.py

```diff
@@ -90,7 +90,7 @@ def start(request):
     identity_item = dict(
```

Comment:
This should use the `viewmodels.MediaItem` instead of `dict` now (which also handles the `details` needing to be a list)

Reply:
On line 98 and 105, additional keys (`links` and `bullets`) are added to the `dict` which are not on `MediaItem`. 

I can actually remove the one on line 98 though since the design no longer has a links on the Login.gov item. I think we said we weren't going to worry about the bullet points for now, and it can just work with the `dict`?

Reply:
Ah... I'm the one that didn't add the `bullets` functionality to the viewmodel when I created it... was waiting for #996 I guess?

Please add it to the viewmodel and use that instead.

Reply:
Can we discuss a spec before I start implementation?

When this came up previously, we weren't sure how `bullets` should be represented on the model (this text would be different per-`EligibilityVerifier`). A verifier could have 0-N bullets that it wants to show.

Reply:
Right so the [template already looks like](https://github.com/cal-itp/benefits/blob/dev/benefits/core/templates/core/includes/media-list.html#L10) (in `dev`):

```html
{% if item.bullets|length %}
    <div class="media-body--items">
        <ul>
            {% for l in item.bullets %}<li>{{ l }}</li>{% endfor %}
        </ul>
    </div>
{% endif %}
```

So I think its taking what you have now and just making it another kwarg on the init, and make it work similar to `details`:

```python
class MediaItem:
  def __init__(self, icon: Icon, details, heading=None, bullets=None):
    if bullets:
      # make sure it's a list similar to details
      # self.bullets = list_of_bullets
```

Reply:
That makes sense 👍 I'll make that change

Reply:
Done in https://github.com/cal-itp/benefits/pull/1013/commits/400bcae738227428a81af871990d059b54e1907e

Reply:
🤘